test/framework/src/main/java/org/elasticsearch/test/hamcrest/ElasticsearchAssertions.java

```diff
@@ -636,7 +636,7 @@ public static void assertVersionSerializable(Version version, Streamable streama
          * a way that sucks less.
          */
         NamedWriteableRegistry registry;
-        if (ESIntegTestCase.isInternalCluster()) {
+        if (ESIntegTestCase.isInternalCluster() && ESIntegTestCase.internalCluster().size() > 0) {
```

Comment:
I don't remember any more to be honest but it was a test managed its own cluster that wasn't available here.